### ISSUE TITLE
{Freeway-Bug} [Adaptive Cards> FlightItinerary.json]: Focus order is not in sequential way in FlightItinerary.json card

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7839]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25182.1010)
Browser: Edge Dev (106.0.1356.0)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2206.2001
Screen Reader: Narrator

**Repro Steps:**

1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ windows+ enter key and press caps lock + space bar key to turn on the scan mode.
3. Navigate to the 'FlightItinerary.json' through tab key and activate it.
4. Now, navigate to 'San Francisco' and 'Amsterdam through up/down arrow key.
5. Observe the issue.

**Actual Result:**
On FlightItinerary.json card, focus order is not in sequential manner. **Focus order is of San Francisco> Amsterdam> SFO> Image to> AMS.**

**Expected Result:**
On FlightItinerary.json card, focus order should be in sequential manner**. Focus order should be in a way San Francisco> SFO>Image to> Amsterdam> AMS**

**User Impact:**
People with cognitive disability will get impacted if focus order is not in sequential way due to which user will have difficulty in making track of the content properly..

**MAS Reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.02

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/104483005/189816358-c2e0a76e-0e05-44b5-a3bf-29afb12cc923.mp4

### Card JSON

```JSON
NA
```

**Comment by PERUAMLLABALAKRISHNA** [comment]
Focus order issue is observed after acting the set due date control.

https://user-images.githubusercontent.com/117892125/201320045-4e66d745-7378-45c9-8d7e-85c6a50aa45a.mp4

**Comment by JeanRoca** [comment]
This issue has been fixed and will be included in our next release. I am closing this issue for now. Once new version is available, if the issue persists please open a new issue to track.

**Comment by vaishali1397** [comment]
Issue has been fixed in below test Environment, hence Adding A11yTTValidated tag.
**Test Environment:**
OS Version: 22H2 (Build 25314. 1010)
Application: Adaptive Card Visualizer (Version 3.2.2303.22003)

**PFA for Reference:** Adaptive cards UWP~ Fixed issue.mp4

https://user-images.githubusercontent.com/104483005/228218906-99e1055e-0db8-4f2e-aea9-2863e30cccea.mp4
